### ISSUE TITLE
Use pretrained model ( CRF ) in Snorkel Labeling functions

### ISSUE THREAD

**Issue by manileo** [snorkel-team/snorkel#1504]
I can't find anywhere that how to use a pretrained model in labeling functions of snorkel for my project of ner data generation. Please Help.

**Comment by brahmaneya** [comment]
Hi @manileo, you can run a pre-trained model on data points as a preprocessor, and use the preprocessor outputs within labeling functions. As an example, you can check out the usage of the pre-trained TextBlob model in the spam tutorial https://github.com/snorkel-team/snorkel-tutorials/blob/master/spam/01_spam_tutorial.py#L423.

**Comment by manileo** [comment]
I have seen example but it seems like different with CRF. I am just predicting the label using the already trained model. And what to mention in preprocessor function and labelling function.
Please correct me , if I am wrong.
